apps/domain/src/main/core/infrastructure/providers/gcp/gcp.py

```diff
@@ -12,7 +12,7 @@ class GCP(Provider):
     """Google Cloud Provider."""
 
     def __init__(self, config: SimpleNamespace) -> None:
-        super().__init__(config)
+        super().__init__(config.root_dir, "gcp")
```

Comment:
The `config.root_dir` needs to be added to the `Config` as it isn't already present.
You can set it to `os.path.join(str(Path.home()), ".pygrid", "apps", str(config.app.name))` for the domain/network case.
The `config.root_dir` would be different when deploying a worker.